### PR TITLE
feat: update docs and upgrade docusaurus dependencies

### PR DESCRIPTION
**Pull request by mkarajohn**

### CODE REVIEW

**Comment by mkarajohn** [comment]
Current dependencies on/for this PR:
* master
  * **PR #74** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/74" target="_blank">Graphite</a> 
    * **PR #75** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/75" target="_blank">Graphite</a> 
      * **PR #76** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/76" target="_blank">Graphite</a> 
        * **PR #77** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/77" target="_blank">Graphite</a> 
          * **PR #78** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/78" target="_blank">Graphite</a> 
            * **PR #79** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/79" target="_blank">Graphite</a> 
              * **PR #80** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/80" target="_blank">Graphite</a> 
                * **PR #81** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/81" target="_blank">Graphite</a> 
                  * **PR #82** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/82" target="_blank">Graphite</a> 
                    * **PR #83** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/83" target="_blank">Graphite</a> 
                      * **PR #85** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/85" target="_blank">Graphite</a> 
                        * **PR #86** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/86" target="_blank">Graphite</a> 
                          * **PR #87** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/87" target="_blank">Graphite</a> 
                            * **PR #88** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/88" target="_blank">Graphite</a> 
                              * **PR #89** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/89" target="_blank">Graphite</a> 
                                * **PR #90** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/90" target="_blank">Graphite</a> 
                                  * **PR #91** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/91" target="_blank">Graphite</a> 
                                    * **PR #92** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/92" target="_blank">Graphite</a> 
                                      * **PR #93** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/93" target="_blank">Graphite</a> 
                                        * **PR #94** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/94" target="_blank">Graphite</a> 
                                          * **PR #95** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/95" target="_blank">Graphite</a> 
                                            * **PR #96** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/96" target="_blank">Graphite</a> 
                                              * **PR #97** <a href="https://app.graphite.dev/github/pr/Orfium/toolbox/97" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Orfium/toolbox/97?utm_source=stack-comment).

File: documentation/docs/api/01-Components/Navigation.mdx [hunk 1]
```diff
@@ -17,41 +17,51 @@ two main parts, the **global navigation bar** and the **local navigation drawer*
 
 #### Global navigation bar
 
-It lists links to all the Orfium products that are available to the user and highlights the one
-that is currently being used. Admin users will also find an extra button that when clicked it will show an admin-only
-navigation menu in the **main navigation section** of the **local navigation drawer**.
+Top-to-bottom, it consists of :
```
Comment (panvourtsis) [inline documentation/docs/api/01-Components/Navigation.mdx:20]: i would expect this info to be on the `docs` section. Anything that explains the how it is handled and affects the developer that uses it must be on the docs. API should be for explicit props explanation.

File: documentation/docs/api/01-Components/Navigation.mdx [hunk 2]
```diff
@@ -17,41 +17,51 @@ two main parts, the **global navigation bar** and the **local navigation drawer*
...
+* the **extra information section**, which can contain various external links to educational or support material. **(Optional)**
 
 You can read more [here](/docs/modules/UI/Navigation%20Component).
 
 :::info
-The extra admin button has a "resume" feature, where clicking on it to exit the admin navigation will take you back to exactly
-where you were before, restoring any previously applied URL params, no matter how many times the URL changed while you were in the
-admin navigation.
-:::
+Exiting the admin-only navigation by clicking the extra admin button will take you back to exactly where you were before
```
Comment (panvourtsis) [inline documentation/docs/api/01-Components/Navigation.mdx:41]: same here, for `docs`

**Review by panvourtsis — APPROVED** [review]

File: documentation/docs/api/01-Components/Navigation.mdx (hunk 1 above)
Comment (mkarajohn) [reply documentation/docs/api/01-Components/Navigation.mdx:20]: As discussed, we will see what the general feedback is on this. Resolving for now.

**Comment by mkarajohn** [comment]
### Merge activity

* **Oct 24, 10:43 AM**: @mkarajohn started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/Orfium/toolbox/97).

**Pull request merged**